packages/codegen/plopfile.js

```diff
@@ -0,0 +1,19 @@
+export default async function (plop) {
+  plop.setGenerator("Story", {
+    description: "Create a story file",
+    prompts: [
+      {
+        type: "input",
+        name: "name",
+        message: "스토리를 작성할 컴포넌트 이름을 입력해주세요",
+      },
```

Comment:
p4;
지금은 어떤 `input`이든 허용되는 것 같아요.
```
  const COMPONENT_PATH = "./src/components";
  const directoryPath = COMPONENT_PATH;
  const files = await fs.readdir(directoryPath);
```

빌드 스크립트에서 작성해주셨던 것처럼 컴포넌트들을 가져와서 이미 있는 컴포넌트명만 입력할 수 있도록 하면 어떨까요?
```suggestion
      {
        type: "input",
        name: "name",
        message: "스토리를 작성할 컴포넌트 이름을 입력해주세요",
        validate: (input) => {
          if (!files.includes(input)) {
            return "만들어진 컴포넌트 이름을 입력해주세요.";
          }
          return true;
        },
      },
```

Reply:
굿굿 좋습니다
다만, 생성되지 않은 컴포넌트의 경우 return 문까지만 작성하면 프로세스가 종료되지 않아서 뒤에 exit()하나만 추가시켜줬습니다~!